renovate.json

```diff
@@ -7,5 +7,11 @@
   "labels": [
     "changelog:dependencies"
   ],
-  "suppressNotifications": ["prEditedNotification"]
+  "suppressNotifications": ["prEditedNotification"],
+  "packageRules": [
+    {
+      "matchUpdateTypes": ["major"],
+      "dependencyDashboardApproval": true
```

Comment:
I'm not sure this is what we want. It sounds like it would propose a major upgrade as an entry in the dashboard issue. But a) we'll never want to trigger it, and b) it might not offer minor upgrades. 

Please link to docs that you found.

Reply:
Ah ok, if the major upgrades never want to be triggered we would change it to below and not need the "packageRules". 
```
  "major": {
    "enabled": false
  }
```
https://docs.renovatebot.com/presets-default/#disablemajorupdates

I had initially thought you wanted the option available in the dashboard.
The documentation for that is https://github.com/renovatebot/tutorial?tab=readme-ov-file#the-dependency-dashboard-includes below the "Rate Limited" image:
```
Since we decided that we want to handle it manually, we will edit configurations and see how the Dependency Dashboard is affected by this change.

In order to limit all major updates to on-demand creation:

    Add this code segment to your renovate.json file:
```

I tested on my fork and with the `dependencyDashboardApproval` the dashboard looks like:
![Screenshot from 2024-06-12 22-04-30](https://github.com/jaegertracing/jaeger/assets/19431067/1f0cefde-acfc-4db9-80b4-808f18564c7d)

And with all major updates disabled:
![Screenshot from 2024-06-12 22-08-49](https://github.com/jaegertracing/jaeger/assets/19431067/8a1bc621-42c4-4cee-a731-d7e332cc1019)

Reply:
we don't want to disable ALL major upgrade across the whole repo, only for those specific docker-compose files used for e2e integration tests with different major versions of external dependencies.

Reply:
Ah ok, sorry I misunderstood. I fixed that, and testing with my fork it created PRs for upgrading minor versions in the docker-compose files, but not major versions (ex: `docker-compose/elasticsearch/v8/docker-compose.yml` went from 8.8.2 to 8.14.1, but `docker-compose/elasticsearch/v7/docker-compose.yml` was unchanged). It did create PRs for upgrading major versions in dockerfiles (ex: `plugin/storage/cassandra/Dockerfile` went from 4.1 to 5.0).